plugins/woocommerce/src/Blocks/BlockTypes/Cart.php

```diff
@@ -246,7 +246,13 @@ protected function enqueue_data( array $attributes = [] ) {
 		$this->asset_data_registry->add( 'isBlockTheme', wc_current_theme_is_fse_theme(), true );
 		$this->asset_data_registry->add( 'activeShippingZones', CartCheckoutUtils::get_shipping_zones(), true );
 
-		$pickup_location_settings = get_option( 'woocommerce_pickup_location_settings', [] );
+		$pickup_location_settings = get_option(
+			'woocommerce_pickup_location_settings',
+			[
+				'enabled' => false,
+				'title'   => __( 'Local Pickup', 'woocommerce' ),
+			]
+		);
```

Comment:
I see that this code is identical throughout various files. What do you think about moving it into a util class and reference to it from the various sections? Something like:
```php
$pickup_location_settings = Some_Utils_Class::get_pickup_location_settings()
```

If we'd ever need to change the logic, we only have to touch one section instead of multiple ones.

Reply:
@nielslange thanks for your suggestion. I think this is a fair suggestion, but it would add another layer of abstraction to climb through when you're trying to figure out where something comes from.

I would be on board if it was complex logic, or this specific piece of code needed to be tested independently, but I don't think that repeating two lines of code a few times across the codebase is a compelling reason to introduce an abstraction.

Reply:
It's rather weird indeed that we need to pass this around in some many places.